django/src/rdwatch/api.py

```diff
@@ -10,4 +12,6 @@
 api.add_router('/performers/', performer_router)
 api.add_router('/regions/', region_router)
 api.add_router('/model-runs/', site_model_router)
+# Note lets not use this, try to implement it in the rdwatch_scoring app
+api.add_router('/scores/', scores)
```

Comment:
I believe we can remove this now, since these routes are now defined in the `api.py` and `urls.py` of the `rdwatch_scoring` app.
```suggestion
```